### ISSUE TITLE
Create a CONTRIBUTING.md

### ISSUE THREAD

**Issue by 0x0elliot** [ArgusOSS/Eye#6]
There are certain standards and guidelines we need to lay down for new contributors to be coming in. A file named `CONTRIBUTING.md` should have the following:

```html
### Rules and guidelines

We welcome contributors with any level of skillset and education to come in and help us put together Eye. We advise you to follow the following set of rules to keep in mind while contributing to our Eye:

1. Please get approval on an issue from a maintainer before starting to work on it.
2. Since we are still in the very nascent stage of our project, We strongly advise you to join our discord server to get involved in the development cycle: https://discord.gg/DJyMGArg8D
3. Please make sure that your PR follows the linting rules that our pre-commit hook enforces.
4. Please make sure that your branch, whether you're a maintainer using this repository itself or a fork should use the format `username/feature` for your branches. So, If my branch where I am working on the auth feature should look like: `0x0elliot/auth`
5. Your PR should be to the `dev` branch.
6. Your PRs should follow the following format: Fixes #{issue number}
7. Please respect your fellow contributors and maintainers. We are all trying our best.
8. If the following rules aren't properly followed through, We have the right to reject your PR.
```

For the time being, It is as simple as that is fine. We will expand our code of conduct and rules as we move ahead in the development stage.

**Comment by 0x0elliot** [comment]
Closed by #8

**Comment by 0x0elliot** [comment]
Incomplete. Re-opened.
